### PR TITLE
Add basic UI template for API

### PR DESCRIPTION
**Pull request by pombredanne**
Borrowed from ScanCode.io

Reference: https://github.com/nexB/vulnerablecode/issues/1465
Thanks-to: Thomas Druez @tdruez

### CODE REVIEW

File: vulnerabilities/templates/rest_framework/api.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{% extends "rest_framework/base.html" %}
+
+{% block style %}
+    {{ block.super }}
+    <style>
+        body {background: none;}
+        .navbar {background: #363636; border-top: 3px solid #3298dc;}
+        .navbar-inverse .navbar-brand {color: white;}
+        .nexb-blue
```
Comment (tdruez) [inline vulnerabilities/templates/rest_framework/api.html:9]: This line seems problematic.

Comment (johnmhoran) [reply vulnerabilities/templates/rest_framework/api.html:9]: Yeah, it does, good eye, looks like somebody started to define the CSS for `.next-blue` and got distracted, never finished.  @pombredanne , where did this code come from?

**Comment by TG1999** [comment]
@pombredanne please have a look when you get a chance and merge it! Thanks!

File: vulnerabilities/templates/rest_framework/api.html (hunk 1 above)
Comment (pombredanne) [reply vulnerabilities/templates/rest_framework/api.html:9]: @tdruez @johnmhoran .nexb-blue is now the name... it is defined in https://github.com/nexB/vulnerablecode/pull/1466/files#diff-9cd3f89490bba2f7d7e1e8174417074cc8f0fbdeaf3b603c0f19f57e0b2e8f2fR15 
is this OK?

Comment (tdruez) [reply vulnerabilities/templates/rest_framework/api.html:9]:
```
.navbar-inverse .navbar-brand {color: white;}
.nexb-blue
.booleanwidget {width: 90%;}
```

You need to remove the `.nexb-blue` line, it's not a correct syntax.

**Review by tdruez — CHANGES_REQUESTED** [review]

**Comment by pombredanne** [comment]
Changes applied. Will merge shortly! @tdruez thanks

**Comment by pombredanne** [comment]
@tdruez all fixed. @TG1999 please review and merge

**Pull request merged**